Microsoft.Toolkit.Win32.UI.Controls/WPF/WebView/WebView.cs

```diff
@@ -0,0 +1,1046 @@
+        {
+            get
+            {
+                VerifyAccess();
+                Verify.IsNotNull(_webViewControl);
+                return _webViewControl?.Version;
+            }
+        }
+
+        private bool WebViewControlInitialized => _webViewControl != null;
+
+        /// <summary>
+        /// Closes the <see cref="T:Microsoft.Toolkit.Win32.UI.Controls.WPF.WebViewHost" />.
+        /// </summary>
+        /// <seealso cref="M:System.IDisposable.Dispose" />
+        public override void Close()
+        {
+            // TODO: Guard IsDisposed
+            UnsubscribeEvents();
+            _webViewControl?.Close();
+            _webViewControl?.Dispose();
+
+            _webViewControl = null;
+            _process = null;
+        }
+
+        /// <summary>
+        /// Gets the deferred permission request with the specified <see cref="WebViewControlPermissionRequest.Id" />.
+        /// </summary>
+        /// <param name="id">The <see cref="WebViewControlPermissionRequest.Id" /> of the deferred permission request.</param>
+        /// <returns><see cref="WebViewControlDeferredPermissionRequest" /> The deferred permission request with the specified <see cref="WebViewControlPermissionRequest.Id" />, or null if no permission request with the specified <see cref="WebViewControlPermissionRequest.Id" /> was found.</returns>
+        public WebViewControlDeferredPermissionRequest GetDeferredPermissionRequestById(uint id)
+        {
+            VerifyAccess();
+            Verify.IsNotNull(_webViewControl);
+            return _webViewControl?.GetDeferredPermissionRequestById(id);
+        }
+
+        /// <summary>
+        /// Navigates the <see cref="WebView"/> to the previous page in the navigation history.
+        /// </summary>
+        /// <returns><c>true</c> if the <see cref="WebView"/> navigation to the previous page in the navigation history is successful; otherwise, <c>false</c>.</returns>
+        public bool GoBack()
+        {
+            VerifyAccess();
+            Verify.IsNotNull(_webViewControl);
+            return _webViewControl?.GoBack() ?? false;
+        }
+
+        /// <summary>
+        /// Navigates the <see cref="WebView"/> to the next page in the navigation history.
+        /// </summary>
+        /// <returns><c>true</c> if the <see cref="WebView"/> navigation to the next page in the navigation history is successful; otherwise, <c>false</c>.</returns>
+        public bool GoForward()
+        {
+            VerifyAccess();
+            Verify.IsNotNull(_webViewControl);
+            return _webViewControl?.GoForward() ?? false;
+        }
+
+        /// <exception cref="InvalidOperationException">When the underlying <see cref="WebView"/> is not yet initialized.</exception>
+        public object InvokeScript(string scriptName)
+        {
+            VerifyAccess();
+            Verify.IsNotNull(_webViewControl);
+            return _webViewControl?.InvokeScript(scriptName);
```

Comment:
@normesta @mcleanbyron @nmetulev  Lots of cases where the method could be invoked by a caller, but the underlying control is not yet initialized (it's VERY asynchronous in WPF). In most cases doing so results in a no-op, but that might be unexpected by the caller? Do we guard these scenarios, provide a property with the initialization state (so callers can check), and throw an exception so it's VERY clear?